src/main/java/core/basesyntax/strategy/impl/AbstractOperationHandler.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.enums.Operation;
+import core.basesyntax.strategy.OperationHandler;
+import java.util.Map;
+
+public abstract class AbstractOperationHandler implements OperationHandler {
```

Comment:
According to the checklist, we should not begin class or method implementation with an empty line. Please remove the redundant empty line at the start of the class definition.